static_src/util/create_styler.js

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames';
+
+function uniqueOnly(value, index, self) {
+  return self.indexOf(value) === index;
+}
+
+export default function createStyler(...args) {
```

Comment:
What happens if there are two styles from separate files that are the same? Or is that something that would've even happen because of CSS Modules?